### DISCUSSION TITLE
extract_words() finding unexisting "blocks"

### DISCUSSION THREAD

**Discussion by Renato-Waldem** [jsvine/pdfplumber#1212] [Ask for help with specific PDFs]
Im having trouble while performing simple extract_words(). It is finding these "blocks" that doesnt have any character int it. Is there any paramenter to avoid those non-characters objects?
![Screenshot 2024-10-09 130806](https://github.com/user-attachments/assets/e28e7661-9a6f-43b3-8263-8747b35356c9)
Thanks in advance

**Comment by jsvine** [comment]
Are you able to share the PDF itself, and the code that you're using?

One possibility is that the PDF *does* contain text there, but that it's just not visible. If you run select-all (something like `Command-a`, depending on the PDF viewer and operating system) on your PDF in a standard PDF viewer and then paste it into a text editor, do you see any text that might correspond to those areas?
